inst/stan/functions/pmfs.stan

```diff
@@ -4,14 +4,17 @@ vector discretised_gamma_pmf(int[] y, real mu, real sigma, int max_val) {
   vector[n] pmf;
   real trunc_pmf;
   // calculate alpha and beta for gamma distribution
-  real c_sigma = sigma + 1e-5;
-  real alpha = ((mu)/ c_sigma)^2;
-  real beta = (mu) / (c_sigma^2);
+  real small = 1e-5;
+  real large = 1e8;
+  real c_sigma = sigma < small ? small : sigma;
+  real c_mu = mu < small ? small : mu;
+  real alpha = ((c_mu) / c_sigma)^2;
+  real beta = (c_mu) / (c_sigma^2);
   //account for numerical issues
-  alpha = alpha <= 0 ? 1e-5 : alpha;
-  beta = beta <= 0 ? 1e-5 : beta;
-  alpha = is_inf(alpha) ? 1e8 : alpha;
-  beta = is_inf(beta) ? 1e8 : beta; 
+  alpha = alpha < small ? small : alpha;
+  beta = beta < small ? small : beta;
+  alpha = is_inf(alpha) ? large : alpha;
```

Comment:
Presumably the same logic applies here?

Reply:
Sounds good! Check f9a3226a157894b03db928fa1886d1a675e471ad !